### ISSUE TITLE
Can't revert or change after applying?

### ISSUE THREAD

**Issue by ArcticJake** [leminlimez/Nugget-Mobile#32]
After applying and the phone restarting, I have tried to disable the 'spoof' for apple intelligence, but can't because it keeps claiming minimixer isn't working. I have tried everything, even Nugget Python... So now I am stuck without Face ID... Please try to fix ASAP.

**Comment by AnUnkn0wnGithubUser** [comment]
Got this problem too, please fix asap

**Comment by maxrisky** [comment]
I have got the same problem

**Comment by xkedude** [comment]
Beta 7 not supported read readme.

**Comment by maxrisky** [comment]
Version 2 supports Beta 7

**Comment by ArcticJake** [comment]
@xkedude latest release says partial support for beta 6 and 7

**Comment by maxrisky** [comment]
> After applying and the phone restarting, I have tried to disable the 'spoof' for apple intelligence, but can't because it keeps claiming minimuxer isn't working. I have tried everything, even Nugget Python... So now I am stuck without Face ID... Please try to fix ASAP. (iOS 18.1 Beta 7)

Well i am trying this. You can try it too meanwhile. Do let me know if it worked for you...

You must wait for it Apple Intelligence (Siri) to be fully downloaded and working, before fixing Face ID/reverting your ProductType.
Get your modified MobileGestalt file from Part 1 (not the backup you made)
Change ProductType key (h9jDsbgj7xIVeIQ8S3/X3Q) back to the correct one for your device (e.g. iPhone14,2 or iPad12,1)
Find out at https://appledb.dev/device-selection/ if you've forgotten
By "back from part 1", you essentially need to keep the key DeviceSupportsGenerativeModelSystems
Now apply this new MobileGestalt file back to your device with Nugget.
Reboot your device, connect to Wi-Fi
Then, wait like 5 minutes again.
It should do some "downloading" for like upto 5 minutes, then the AI (Siri) should start working again!

**Comment by ArcticJake** [comment]
@maxrisky if you're referring to Nugget Python, it doesn't work for beta 7 yet...

**Comment by maxrisky** [comment]
@ArcticJake i am trying through MiskaX 2.2

**Comment by ArcticJake** [comment]
@maxrisky ah, i'll try that then

**Comment by maxrisky** [comment]
@ArcticJake Any luck?

**Comment by ArcticJake** [comment]
@maxrisky nope

**Comment by maxrisky** [comment]
@ArcticJake I am downgrading to 18.1 beta 4

**Comment by ArcticJake** [comment]
@maxrisky im just gonna wait until a patch comes out

**Comment by maxrisky** [comment]
@ArcticJake Apple intelligence is working good for you? What features are accessible? Can you access the writing tools?

**Comment by ArcticJake** [comment]
@maxrisky apple intelligence works (obviously besides as advertised lol) i'll check the features currently accessible in a moment

**Comment by ArcticJake** [comment]
@maxrisky no writing tools 😭

**Comment by maxrisky** [comment]
@ArcticJake It's just apple's stunt to make people buy the newer iphones. Same features can run even on iphone 11s :D

**Comment by ArcticJake** [comment]
@maxrisky yeah, it's basically just normal siri just with a new 'ui' i suppose. (using an Xr right now)
